commit 23dfcd45668d2076dfceeeb96d80a42b06240fe6
Author: Nathan Scott <nathans@redhat.com>
Date:   Thu Mar 20 14:05:30 2014 +1100

    Fix build on RHEL5 i386 where libpcp has unexpected statics
    
    $ ./check-statics
    rtime.o: d startend_relative_terms.9369 : Error: additional symbol

diff --git a/src/libpcp/src/check-statics b/src/libpcp/src/check-statics
index 086167786965..7f04ee771e34 100755
--- a/src/libpcp/src/check-statics
+++ b/src/libpcp/src/check-statics
@@ -315,7 +315,7 @@ rtime.o
     r ?ampm			# const (MinGW)
     r ?months			# const (MinGW)
     r ?wdays			# const (MinGW)
-    s ?startend_relative_terms.[0-9]*	# const
+    [ds] ?startend_relative_terms.[0-9]*	# const
 sortinst.o
 spec.o
 store.o
